locale/en/get-involved/index.md

```diff
@@ -42,3 +42,4 @@ The Node.js community is large, inclusive, and excited to enable as many users t
 - [Spanish language Facebook group for Node.js](https://www.facebook.com/groups/node.es/)
 - [Spanish language community](http://nodehispano.com)
 - [Việt Nam Node.js community](http://nodejs.vn)
+- [NodeJS Discord community](https://discord.gg/Uaq5RHu) - (unofficial)
```

Comment:
```suggestion
- [Node.js Discord community](https://discord.gg/Uaq5RHu) - (unofficial)
```

I don’t have an opinion on whether this should be merged or not yet, but if we do, I’d go with the spelling of “Node.js”.

Reply:
Yes, `Node.js` that is the spelling to use as instructed by the Foundation and also the core repo style guide. 👍